spaconclj/spacon/src/spacon/components/user.clj

```diff
@@ -2,21 +2,22 @@
   (:require [com.stuartsierra.component :as component]
             [io.pedestal.interceptor.helpers :refer [defhandler]]
             [spacon.http.intercept :as intercept]
+            [spacon.http.response :as response]
             [spacon.models.user :as user]
             [spacon.http.auth :refer [check-auth]]
             [clojure.spec :as s]))
 
-(defhandler get-all-users [request]
-  {:status 200 :body (map user/sanitize (user/find-all))})
+(defn get-all-users [context]
+  (response/success (map user/sanitize (user/find-all)) "Content-Type" "application/json"))
```

Comment:
Can you remove the header here?  Won't it be added further down the chain anyway?